src/storage/patch/RdfStorePatcher.ts

```diff
@@ -0,0 +1,15 @@
+import type { Store } from 'n3';
+import type { Patch } from '../../http/representation/Patch';
+import type { ResourceIdentifier } from '../../http/representation/ResourceIdentifier';
+import { AsyncHandler } from '../../util/handlers/AsyncHandler';
+
+export interface RdfStorePatcherInput {
+  identifier: ResourceIdentifier;
+  patch: Patch;
+  store: Store;
```

Comment:
Add tsdoc for all parameters. Specifically important for the store so it is mentioned that the `RdfStorePatcher` is allowed to modify the contents of the store.